pom.xml

```diff
@@ -604,7 +604,7 @@ THE SOFTWARE.
         <configuration>
           <signature>
             <groupId>org.codehaus.mojo.signature</groupId>
-            <artifactId>java1${java.level}</artifactId>
+            <artifactId>java16</artifactId>
```

Comment:
Enforcer would fail if we leave `${java.level}` as-is. If you really want to do it properly we'd have to create two variables. I don't think we need that, do we?